packages/amplication-design-system/src/components/LimitationDialog/LimitationDialog.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
```

Comment:
thus is not a design system component. it should be part the application. it is too specific